### PR TITLE
Add test scripts for pull-tf-minigo-cc

### PR DESCRIPTION
**Pull request by chsigg**
This makes  testing similar to the python tests and allow changing the testing setup as part of a PR.

The script does the steps currently hardcoded in
https://github.com/kubernetes/test-infra/blob/master/config/jobs/tensorflow/minigo/minigo.yaml,
which would need to call this script instead in a follow-up change.

The script also checks the C++ formatting.

### CODE REVIEW

File: cc/test.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  exit 1
+fi
+
+bazel test //cc:all --test_output=errors --compilation_mode=dbg --define=board_size=9 || {
+  echo >&2 "--------------------------------------"
+  echo >&2 "The tests did not pass successfully!"
+  exit 1
+}
+
+bazel test //cc:all --test_output=errors --compilation_mode=dbg --define=board_size=19 || {
```
[the remark this answers is not in the document]
Comment (chsigg) [reply cc/test.sh]: Done.

**Comment by chsigg** [comment]
Thanks a lot Tom for the quick turn around!

**Comment by amj** [comment]
LGTM, we'll ping @cjwagner when we're ready to land the other change in kubernetes/test-infra

File: testing/setup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# limitations under the License.
+
+# Simple shell script to lint files and run the tests. Could be helpful for
+# users, but largely for automation.
+#
+# NOTE! If this file changes/moves, please change
+# https://github.com/kubernetes/test-infra/blob/master/config/jobs/tensorflow/minigo/minigo.yaml
+
+# Copy prebuilt TensorFlow from image.
+rsync -ahv /app/cc/tensorflow/ ./cc/tensorflow/
```
Comment (cjwagner) [inline testing/setup.sh]: So the ProwJob entrypoint command will now be something like `testing/setup.sh && cc/test.sh`, correct?

**Pull request merged**